dashboard/settings.py

```diff
@@ -25,35 +23,37 @@
 PROJECT_ROOT = os.path.abspath(
     os.path.join(os.path.dirname(__file__), ".."),
 )
+with open(os.getenv("ENV_FILE", "/code/config/env.json")) as f:
```

Comment:
In OpenShift environment we might need to configure secrets added to that path since the path is Hardcoded

Reply:
Yeah that's what this ENV_FILE environment variable is for. We can change the location of this json file with an environment variable. 

There are still going to be a few environment variables such as the openshift ones (about the build) and MYSQL needs them for the MYSQL Docker image, but not many anymore.

Reply:
I missed that part, sorry!